libs/auth-api-lib/seeders/20210322163513-applications-resources.js

```diff
@@ -63,7 +63,7 @@ const api_scope_user_claims = [
   },
 ]
 
-const client_id = 'island.is_1'
+const client_id = 'island-is_1'
```

Comment:
this is still modified, should be `island-is-1`

Reply:
Sorry... aðeins of fljótfær :(